docs/compile_guide.md

```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-This guide provides instructions for developers to build and run Harbor from source code. 
+This guide provides instructions for developers to build and run Harbor from source code.
 
 ## Step 1: Prepare for a build environment for Harbor
```

Comment:
The build host requires Python, .....
change to
The build environment requires Python,